docs/development/_index.md

```diff
@@ -54,6 +54,109 @@ options can be appended to the main command as arguments.
 make image
 ```
 
+
+### Running the docker image
+
+With docker you can run the Headlamp image. Note, the mount arguments add folders that are referenced in the ~/.kube folders - you may need to add other folders if your config has more.
+
+```bash
+docker run --network="host" -p 127.0.0.1:4466:4466/tcp --mount type=bind,source="/home/rene/.minikube",target=$HOME/.minikube --mount type=bind,source="$HOME/.kube",target=/root/.kube quay.io/kinvolk/headlamp:latest /headlamp/server -html-static-dir /headlamp/frontend -plugins-dir=/headlamp/plugins
+```
+
+If you just made an new docker image you can run it like this:
+
+```bash
+$ DOCKER_IMAGE_VERSION=development make image
+...
+Successfully tagged kinvolk/headlamp:development
+
+$ docker run --network="host" -p 127.0.0.1:4466:4466/tcp --mount type=bind,source="/home/rene/.minikube",target=$HOME/.minikube --mount type=bind,source="$HOME/.kube",target=/root/.kube kinvolk/headlamp:development /headlamp/server -html-static-dir /headlamp/frontend -plugins-dir=/headlamp/plugins
+```
+
+Then go to https://localhost:4466
+
+
+### Minikube "incluster"
+
+This is if you want to use Headlamp incluster, and test it locally on minikube with a local docker image.
+
+#### docker image in the minikub docker environment
+
+First we have to make the image on the minikube docker environment.
+
+```bash
+$ eval $(minikube docker-env)
+$ DOCKER_IMAGE_VERSION=development make image
+```
+
+#### Create a deployment yaml.
+
+```bash
+$ kubectl create deployment headlamp -n kube-system --image=kinvolk/headlamp:development -o yaml --dry-run -- /headlamp/server -html-static-dir /headlamp/frontend -in-cluster -plugins-dir=/headlamp/plugins > minikube-headlamp.yaml
+```
+
+To use the local docker image we change the imagePullPolicy to Never.
+
+```yaml
```

Comment:
We have a Kubernetes YAML file, maybe we should reuse it here and just have users change the needed parts (maybe through `sed` as we also have in other docs). Otherwise we have to keep this up to date, not a big deal, just seems like a good idea IMO.

Reply:
I couldn't find a simple expression to add the imagePullPolicy line to the yaml in the right spot nicely without a complex confusing regex. Also that command isn't available as a command line argument. I'm not familiar with the documentation system, but some other ones can include from files/urls, but couldn't see how to do that either. With that, maybe including this yaml is the better way.